vertx-mssql-client/src/test/java/io/vertx/mssqlclient/tck/MSSQLBinaryDataTypeEncodeTest.java

```diff
@@ -57,5 +57,13 @@ public void testDecimal(TestContext ctx) {
     //TODO do we need wrapped type?
     super.testDecimal(ctx);
   }
+  
+  @Test
+  @Ignore // TODO: Investigate this issue and enable the test 
+  // (unfortunately it wasn't as simple as removing the NOT NULL constraints in init.sql)
+  @Override
+  public void testNullValues(TestContext ctx) {
+    super.testNullValues(ctx);
+  }  
```

Comment:
CC @BillyYccc for this change. I'm adding `testNullValues` to the common TCK in this PR, but it fails for MSSQL. Can you please investigate this when you have time?

Reply:
I think it's not supported in the client for now.